### DISCUSSION TITLE
Sujet / Thema 7 –– NGDI-Karte und Glossar / Carte et glossaire INDG (Let's map the NGDI! 🚀)

### DISCUSSION THREAD

**Discussion by rastrau** [GeoUnconference/discussions#31] [Ideen / Idées]
Mich würde interessieren, welche Systeme / Angebote / Bestandteile wir als Teil der NGDI auffassen? In einem zweiten Schritt würde ich ganz gerne erheben, welche der genannten Komponenten (wie gut) bekannt sind.

** Spreadsheet** 
https://docs.google.com/spreadsheets/d/11i--5aUhEIBC86WHcaRqOWzAU7kq55BykAxkLILC5oE/edit?usp=sharing bitte ergänzen

Aus einem anderen Thread mit einer Diskussion mit u.a. @mizehnder und noch weiter angereichert sehe ich folgende mögliche "Kandidaten" (in grob absteigender "Zugehörigkeit" zur NGDI):

# Portale
- Geoportale (inklusive Services):
  - des Bundes (inklusive Themenportale verschiedener Bundesämter): geo.admin.ch 
  - der (einzelnen) Kantone
  - der Kantone (gemeinsam) bzw. von KGK: https://geodienste.ch
  - der einzelnen Städte / Gemeinden
  - von Städten / Gemeinden (gemeinsam): z.B. https://raumdatenpool.ch
  - von Public Private Partnership o.ä.:
    - mit amtlichen Daten: eventuell https://www.geogr.ch (?). Ist diese Kategorisierung korrekt? Gibt es weitere Beispiele?
    - mit amtlichen und privaten Daten: z.B. https://viageo.ch, https://plans-reseaux.ch
  - von Privaten: 
    - mit amtlichen Daten: z.B. https://giszh.ch. Ist diese Kategorisierung korrekt? Gehört dies zur NGDI? Gibt es weitere Beispiele?
    - mit privaten Daten: z.B. https://swisspost.opendatasoft.com, https://data.sbb.ch Gehört dies zur NGDI? Gibt es weitere Beispiele?
- Daten-Repository des Bundes: https://data.geo.admin.ch
- https://www.cadastre.ch
- https://beta.swissgeol.ch
- Metadatenportal geocat.ch
- Metaportale:
  - geodatenbezogene Teile von opendata.swiss (?)
  - geodatenbezogene Teile von opentransportdata.ch (?)

# Rechtlichen Grundlagen
- https://geobasisdaten.ch,
- https://geolex.ch
- GeoIG und GeoIV
- KGeoIGs und KGeoIVs
- GGeoIGs und GGeoIVs
- weitere Gesetzgebungen: AV, Tarife, ÖREBK, ...

# Gremien
- KGK-CGC
- swisstopo KOGIS
- SOGI-OSIG

# Interlis
- Website: www.interlis.ch 
- Modell-Repositories:
  - http://models.interlis.ch
  - http://models.geo.admin.ch
  - http://models.geo.kgk-cgc.ch
  - kantonale / kommunale / weitere
- Software:
  - ili2c
  - umleditor
  - iG/Check
  - ilivalidator
  - ili2fme
  - ili2db
- http://geostandards.ch

# Weitere (Sind diese Teil der NGDI? Wie wären sie zu kategorisieren?)
- Betreiber regionaler Geodateninfrastrukturen (RGDI) (@mizehnder): z.B. https://asit-asso.ch
- https://www.geowebforum.ch
- https://sosm.ch
- Dokumentation GeoAdmin API: https://api3.geo.admin.ch
- Hashtag #swissgis: https://twitter.com/search?q=%23swissgis
- https://groups.google.com/g/geoadmin-api?pli=1
- https://api.geo.admin.ch
- https://ld.geo.admin.ch

# Standards? Welche?

# Strategien
- [Strategie Geoinformation Schweiz](https://www.geo.admin.ch/de/ueber-geo-admin/leistungsauftrag/strategie-und-umsetzung.html)
- kantonale Strategien
- kommunale / städtische Strategien (welche gibt es hier?)
- weitere?

# Kanäle?
- [@KGK_CGC](https://twitter.com/KGK_CGC)
- [@swiss_geoportal](https://twitter.com/swiss_geoportal)
- [@swisstopo_fr](https://twitter.com/swisstopo_fr)
- [@swisstopo](https://twitter.com/swisstopo)
- [@SITGeneve](https://twitter.com/SITGeneve)
- [@GISKtAI](https://twitter.com/GisKtai)
- [@GISUster](https://twitter.com/GISUster)
- [@sogiosigch](https://twitter.com/sogiosigch)
- [@GISKantonZug](https://twitter.com/GISKantonZug)
- [@_sogis](https://twitter.com/_sogis)
- [@GeoPortalBS](https://twitter.com/GeoPortalBS)
- [@GeoKtZH](https://twitter.com/GeoKtZH)
- [@ThurGIS](https://twitter.com/ThurGIS)
- [@geoszch](https://twitter.com/geoszch)
- [@GL_Geoportal](https://twitter.com/GL_Geoportal)
- weitere? (here https://docs.google.com/spreadsheets/d/1neDdHluruDonveHNv-oJwH0xYyMvtnAVCI10ms6Z3vk/edit?usp=sharing) 

# Veranstaltungen
- GEOSummit
- SOGI Feierabendforen
- Spirgarten-Treffen
- Journée Romande de la Géoinformation
- GeoBeer
- GeoUnconference
- Opendata.ch-Konferenzen

# Anwenderinnen- und Anwenderforen?
- [QGIS Usergroup Switzerland](https://www.qgis.ch/en)
- [Esri Userforum Schweiz](https://www.esriuserforum.ch/)
- [Hexagon Swiss Chapter GeoForum](http://www.geo-forum.ch/)

# Weitere Kategorien?

Fragen: 
1. Welche Kategorien oder Instanzen innerhalb Kategorien sind Teil der NGDI? ... oder nicht Teil der NGDI? 
2. Welche weiteren NGDI-Kategorien gibt es noch?
3. Welche zusätzlichen Beispiele (Instanzen innerhalb NGDI-Kategorien) gibt es noch?

**Comment by mizehnder** [comment]
Bei den Geoportalen fehlt mir noch folgende Ergänzung:

- Geoportale (inklusive Services) mit amtlichen **und privaten** Daten, die durch eine Public Private Partnership o.ä. angeboten werden: Das wären für mich dann z.B. die Angebote von ASIT wie https://viageo.ch/ oder https://plans-reseaux.ch/

- Diese Angebote und Struktur von ASIT entsprechen recht gut meinem Bild einer NGDI im Kleinen, sprich einer regionalen Geodateninfrastruktur (RGDI) und müssen Teil der NGDI sein.

Nicht Teil der NGDI sind für mich die Kategorien "Veranstaltungen" und "Anwenderforen", diese sind eher Teil einer Nationalen Geo-Community aber nicht der **I**nfrastruktur.

Ergänzen würde ich noch die Kategorie der "rechtlichen Grundlagen" mit den Instanzen https://geobasisdaten.ch,
https://geolex.ch, GeoIG, GeoIV, etc.

**Reply by rastrau** [reply]
Danke, Mirjam! Ich habe diese oben noch ergänzt bzw. entsprechend einsortiert und die Reihenfolge der Kategorien von subjekt "definitiv Teil der NGDI" bis "vielleicht / eher nicht Teil der NGDI" geändert.

**Reply by davidoesch** [reply]
Veranstaltungen und Anwenderforen als " Kontaktnetz" sind ein Zentraler Teil der NGDI... siehe [Umsetzungskonzept ](https://cms.geo.admin.ch/www.e-geo.ch/archives/pdf_egeo_de/umsetzungskonzeptbroschuere2004de.pdf) Seite  3
"Bestehende nationale Informationsgemeinschaften werden intensiviert und in spezifischen Themenbereichen durch weitere
aufzubauende Gemeinschaften ergänzt." -> Veranstaltungen etc wie zb geounconference geobeer gehören auch dazu
![image](https://user-images.githubusercontent.com/4577727/137309381-f9c88d2a-e66b-493f-b112-90cb54260642.png)

**Reply by mizehnder** [reply]
@davidoesch gehören Veranstaltungen und Anwenderforen aus deiner Sicht bloss zur NGDI dazu, weil etwas davon im Umsetzungskonzept aus dem Jahr 2004 steht oder gibt es noch weitere Gründe dafür?

**Reply by davidoesch** [reply]
Die neun Handlungsfelder decken alle Aufträge aus der Botschaft zum GeoIG ab. Weiter Gründe: "Kontaktnetz" ist ein wichtiger Bestandteil der NGDI--wenn nicht sogar der Wichtigste: es sind die Akteure (Nutzer, Datenherren, Intermediäre) die die Inwertsetzung Daten vorantreiben. Darum: nicht nur auf "Technik" reduzieren----

**Comment by davidoesch** [comment]
[Edit] **Thread für Sunburst-Darstellungen und Diskussionen**

**Reply by davidoesch** [reply]
https://app.rawgraphs.io/ -> sunburst

**Reply by davidoesch** [reply]
made a spreadsheet for sunburst https://docs.google.com/spreadsheets/d/11i--5aUhEIBC86WHcaRqOWzAU7kq55BykAxkLILC5oE/edit?usp=sharing

used in https://app.rawgraphs.io (.rawgraph config file ) open 

[V2.zip](https://github.com/GeoUnconference/discussions/files/7367517/V2.zip)
to have the live link in rawsgraph and config already set up

![V1](https://user-images.githubusercontent.com/4577727/137767008-e485bc66-89d6-4ddb-a548-24a1c4cb8f0c.png)

**Reply by rastrau** [reply]
Updated: 

![viz (2)](https://user-images.githubusercontent.com/4510815/137847961-f928d243-ae92-4380-9317-44685cfb728e.png)

**Comment by davidoesch** [comment]
Version 3
[viz3.zip](https://github.com/GeoUnconference/discussions/files/7416735/viz3.zip)
![viz3](https://user-images.githubusercontent.com/4577727/138847874-0a1363cb-e6ae-4faf-8cfc-f80e75afc34e.png)

**Reply by ratnanil** [reply]
Sieht toll aus! Auf welcher library basiert das? ist das gar webtauglich und man könnte es irgendwo platzieren?

**Reply by rastrau** [reply]
Im Moment ist das mal ein Prototyp. Je nach inhaltlichem Fokus wäre vielleicht auch eine Treemap und/oder eine Netzwerkdarstellung interessant. Vielleicht arbeiten wir weiter daran an der GeoUnconference 3 im Frühling, @davidoesch @ratnanil und andere Interessierte?

**Reply by ratnanil** [reply]
Gerne!

**Comment by davidoesch** [comment]
https://app.rawgraphs.io/ -> sunburst

**Reply by rastrau** [reply]
Die Antwort erfolgte out-of-thread. Ping @ratnanil, falls Du das ev. nicht gesehen hast.

**Reply by ratnanil** [reply]
Ich konnte auch an der letzten Unconference nicht dabei sein, schade! Ich habe aber die Daten von @davidoesch mit plotly in ein Sunburst Plot gerendert. Dies hat den Vorteil, dass die Visualisierung interaktiv ist (mit klick auf ein Parent *node* werden nur die jeweiligen *children* dargestellt). Das Resultat ist hier ersichtlich (die wenigen Zeilen Code sind [in einem Fork dieses Repos](github.com/ratnanil/geounconference---discussions)):

https://ratnanil.github.io/Geounconference---discussions/thema-7/
<details>
<summary>PS:</summary>
weil ich einen Apex Domain eingerichtet habe, wechselt der URL auf ratnaweera.xyz
</details>

**Reply by davidoesch** [reply]
Das ist wirklich eine gute interaktive Darstellung!
Auch dank github einfach nachführbar

**Comment by rastrau** [comment]
The results of the group working on this topic during GeoUnconference 4 (@p1d1d1 and Pia) are [in this Miro Board](https://miro.com/app/board/uXjVPOcTByI=/?share_link_id=961907439526) ([detailled screenshot here](https://raw.githubusercontent.com/GeoUnconference/geounconference/main/assets/geounconference-4/topic-7-miro-board-detailled.png)):

[![](https://raw.githubusercontent.com/GeoUnconference/geounconference/main/assets/geounconference-4/topic-7-miro-board.png)](https://miro.com/app/board/uXjVPOcTByI=/?share_link_id=961907439526)

The group has shifted their topic somewhat towards [theme 1, Einfacher Zugang / Accés simple](https://github.com/GeoUnconference/discussions/discussions/3).

**Comment by davidoesch** [comment]
Dear group : a POC https://github.com/davidoesch/geoservice_harvester_poc of a "Service Harvester" has been established and we will bring it an operational level together with an FHNW project ....

**Comment by davidoesch** [comment]
you can now search 21'000 datasets from swiss gov and drag and drop the desired Layer /Dataset as WMS WMTS WFS directly to your QGIS or ArcGISPro Desktop Client, feedback welcome : test it: https://davidoesch.github.io/geoservice_harvester_poc/
